src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -193,11 +202,15 @@ public void stop() throws ConnectException {
     @Override
     public void put(Collection<SinkRecord> records) throws ConnectException {
         for (SinkRecord record : records) {
+            log.warn("record to topic:" + record.topic());
```

Comment:
this will be way verbose (there are many calls to put), I wouldn't log here

Reply:
u r right - it was for debugging ..